lib/streamlit/caching/cache_utils.py

```diff
@@ -123,20 +143,27 @@ def get_or_create_cached_value():
             value_key = _make_value_key(cached_func.cache_type, func, *args, **kwargs)
 
             try:
-                return_value = cache.read_value(value_key)
+                value = cache.read_value(value_key)
```

Comment:
value is a dumb name, both because it has messages and because it leads to the following `value.value`